src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -233,10 +235,11 @@ public void canOverrideRenewalWhenItemIsNotRenewableAndNewDueDateIsSpecified() t
     final IndividualResource loanPolicy = loanPoliciesFixture.create(nonRenewablePolicy);
     UUID nonRenewablePolicyId = loanPolicy.getId();
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(loanPolicy)`?

Reply:
Done.